content/handbook/solutions-architects/playbooks/pov/ai.md

```
@@ -0,0 +1,728 @@
+- 制約。次の質問への回答に応じて、一部の AI 機能はすぐには利用できないかもしれません。あるいはプロジェクトのサブセットには適用されない場合もあります。
+  - Self-managed か Dedicated か? Cloud licensing は可能か? Premium か Ultimate か?
+  - セキュリティとプライバシー: 自社インフラ外への API コールを許可するか? プロプライエタリコードの送信も含めて?
+
+技術評価に大きな影響を与えた顧客インプットの例:
+
+- "Code suggestions は私たちには有用ではありません。この機能はアクティベートしたくありません。代わりに、既存のアプリケーションを最大限に活用するために GitLab Duo を活用したいと考えています: ドキュメント化、リファクタリング、テスト追加、インナーソーシングなど。"
+- "AI 機能は、データがネットワーク外に出ないセルフホストのカスタム AI モデルに接続される場合のみ使用したいと考えています"
+
+#### DAP Trial ディスカバリー質問 {#dap-trial-discovery-questions}
+
+以下の質問は[上記の一般的なディスカバリーガイダンス](#discovery---adapt-to-the-customers-context)を補完するもので、DAP のクレジットベーストライアルに特化しています。高価値ユースケースを特定し、ペインポイントを理解し、採用のケースを構築するために使用してください。より広範な販売ディスカバリー質問（GitHub、Azure DevOps、Bitbucket に対する競合ポジショニングを含む）については、[Qualification Questions](/handbook/sales/qualification-questions/) ページを参照してください。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)